suseconnect/migration.go

```diff
@@ -590,6 +601,19 @@ func migrateSystem(migration connect.MigrationPath, forceDisableRepos bool) (str
 		if interrupted {
 			return baseProductVersion, fmt.Errorf("%s: %v", msg, ErrInterrupted)
 		}
+		// mark OLD service as migrated to skip it from cleanup step below
+		migratedServices.Add(service.ObsoletedName)
+	}
+	// remove SUSE services which don't have migration available (bsc#1161891)
+	for _, s := range systemServices {
+		if isSUSEService(s) && !migratedServices.Contains(s.Name) {
+			msg := "Removing service " + s.Name
+			VerboseOut.Println(msg)
+			err := connect.MigrationRemoveService(s.Name)
+			if err != nil {
+				return baseProductVersion, err
```

Comment:
should this be
`return baseProductVersion, fmt.Errorf("%s: %v", msg, err)`

Reply:
good catch!
done